packages/cli/src/util/update-notifier/index.ts

```diff
@@ -0,0 +1,103 @@
+import semver from 'semver';
+import XDGAppPaths from 'xdg-app-paths';
+import { dirname, resolve as resolvePath } from 'path';
+// import { fileURLToPath } from 'url';
```

Comment:
Remove?

Reply:
Removed. You'll need to add it back when you port it to ESM.